source/includes/_contacts.md

```diff
@@ -460,7 +460,7 @@ sort_by_field | Possible values: "contact_last_activity_date", "contact_email_la
 sort_ascending | Possible values: true or false
 page | which page to return. Defaults to 1
```

Comment:
Sentence case for description: First letter should be capital
`which page to return` -> `Which page to return`